tests/e2e/specs/shopper/active-filters.test.ts

```diff
@@ -113,18 +113,21 @@ describe( 'Shopper → Active Filters Block', () => {
 				}
 			);
 
+			await page.waitForSelector(
+				selectors.frontend.productsList + '.is-loading',
+				{ hidden: true }
+			);
+
 			await expect( page ).toClick( 'label', {
 				text: 'In stock',
 			} );
 
-			await page.waitForSelector( block.class );
+			await page.screenshot( {
+				path: 'aaa.png',
+				fullPage: true,
+			} );
 
-			await expect( page ).toMatchElement(
-				selectors.frontend.activeFilterType,
-				{
-					text: 'Stock Status',
-				}
-			);
+			await expect( page ).toMatch( 'Stock Status' );
```

Comment:
I like that you have refactored the result expectation out (although I probably wouldn't have put it as a property of the `block` object, as it seems a bit confusing), what do you think about getting this string out as well? I'm not sure what it is, but something like `FILTER_TITLE` ?

Reply:
Thanks for pointing this out. [I extracted more string](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/42e552d5f28c64f6e675f5a0a3eb145d934a3088/tests/e2e/specs/shopper/active-filters.test.ts/#L51-L54). Do you think that we should put these constants in `utils/constants.js`. I think that for these one it doesn't make sense because they are not for general purpose, but just for this block.

Reply:
I agree with you. If they are only for this block, it's ok to have them scoped in this file. If you can see some other test reusing them, it'd be nice to have them in the constants file.